server/monitor/src/main/resources/resources/js/tables.js

```diff
@@ -184,7 +184,7 @@ function populateTable(ns) {
 
         var row = [];
         row.push(createFirstCell(val.tablename,
-            '<a href="/tables/' + val.tableId + '">' + val.tablename + '</a>'));
+            '<a href="/tables/' + atob(val.tableId.utf8) + '">' + val.tablename + '</a>'));
```

Comment:
Good find! While this is one possible workaround, I think the core of this problem is actually that the Table.ID object is being stored in the first place. I think the real problem is that `org.apache.accumulo.monitor.rest.tables.TableInformation` is storing a `Table.ID` instead of its canonical string. Since the point of this class is to create a serialized version of the table id, I think it should store it as a String in that class. Fixing it there, will also fix the JSON objects in the REST calls, wherever it is used, rather than just in this code.